tests/unit/Lookup/RestrictedEntityLookupTest.php

```diff
@@ -101,6 +105,13 @@ public function testGetEntity_exception() {
 		$lookup->getEntity( new ItemId( 'Q4' ) );
 	}
 
+	public function testGetEntity_exceptionDisabledEntityType() {
+		$lookup = new RestrictedEntityLookup( $this->getEntityLookup(), 200 );
+		$lookup->setDisabledEntityTypes( [ 'item' ] );
+		$this->setExpectedException( EntityLookupException::class );
```

Comment:
This is PHPUnit syntax that is no longer in any supported version. Not an issue with this PR though I do want to raise the question if this component can use a version of PHPUnit that is supported.

Reply:
This is very much unrelated to anything going on here. Currently this syntax is used 500+ times in all Wikibase related codebases combined. When we get the chance to switch to newer versions (which will happen soon), this is mostly a trivial string replace (`setExpectedException` → `expectException`, except we test for exception messages, which we barely do).

Reply:
So to answer the original question:
As long as Wikibase supports PHP 5.5 we are bound to PHPUnit 4.8. Relatively soon (although I think rather months than weeks) we might be able to go up to PHP 5.6 and PHPUnit 5. But it is not really great news I think, as when we do it, PHPUnit 5 will already be EOL. Full switch to PHP 7 is going to take some more (hopefully before PHP 7 and PHPUnit 6 are ancient).

So sadly, I think the answer to what @JeroenDeDauw is: not really.